### ISSUE TITLE
System.TimeoutException

### ISSUE THREAD

**Issue by stevechin1986** [Microsoft/CRM-Performance-Toolkit#10]
Hello.
I'm getting a few System.TImeoutException errors and Fault Exceptions. (See Attached)

I'm wondering if anyone has successfully circumvented these issues, and how?

Thanks!

[Doc1.docx](https://github.com/Microsoft/CRM-Performance-Toolkit/files/44468/Doc1.docx)

**Comment by jeffreyzhzhang** [comment]
Yes, we have customers using the toolkit successfully to run their load test. It is hard to tell what went wrong based on the information you provided. Are you currently working with anyone from Microsoft?

**Comment by stevechin1986** [comment]
I am working with Microsoft, but I'm having some trouble getting any traction. 
How would you suggest for me to best get in contact with either yourself or somebody else who is knowledgeable with this Performance Toolkit?

Currently in touch with a Partner Technical Consultant (Dynamics CRM) at Microsoft.
There are advisory hours (potentially even premier support) that are available if that is an issue.

Thank you!
Steve
